hbase-oss/pom.xml

```diff
@@ -0,0 +1,234 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License. See accompanying LICENSE file.
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>org.apache.hbase</groupId>
+    <artifactId>hbase-filesystem</artifactId>
+    <version>1.0.0-alpha1-SNAPSHOT</version>
+    <relativePath>../pom.xml</relativePath>
+  </parent>
+  <artifactId>hbase-oss</artifactId>
+  <name>Apache HBase / Object-Store Semantics Module</name>
+  <description>
+    This module provides atomic file-system semantics required by HBase when
+    running on object-store based FileSystem implementations that can not
+    natively offer those semantics. It does this by locking, so operations may
+    not be fast, but should be transactional.
+  </description>
+  <packaging>jar</packaging>
+
+  <properties>
+    <fs.hboss.sync.impl>org.apache.hadoop.hbase.oss.sync.ZKTreeLockManager</fs.hboss.sync.impl>
+    <aws-java-sdk.version>1.11.525</aws-java-sdk.version>
+  </properties>
+
+  <profiles>
+    <profile>
+      <id>null</id>
+      <activation>
+        <property>
+          <name>null</name>
```

Comment:
What's gained by having the property activation? Is there some context in which folks can't pass `-Pnull` but can pass `-Dnull`? If the goal is to ensure only a single sync impl is used, better to do something like `-Dhboss.lock.manager=null`

Reply:
Force of habit from S3Guard, where the most common options could be activate by -D or -P for convenience. No objection to removing.